### ISSUE TITLE
Performance gap on VOC

### ISSUE THREAD

**Issue by zbf1991** [jbeomlee93/RIB#5]
Hi, I am trying to reproduce your results on VOC, the mIoU of the final pseudo label is 67.1\%, which is similar to your reported results in the paper. But when I use those generated pseudo labels to train deeplab-v2 (initializing with image-net pretrained model), the final performance with CRF can only reach 65.3\%.

I am following the same setting mentioned in your another [issue](jbeomlee93/AdvCAM/issues/9), with batch size:10, iter_max: 30000, lr: 2.5e-4, dataset scale for training: [0.5, 0.75, 1.0, 1.25, 1.5, 1.75, 2.0], and the balanced CEloss.

I want to know whether your config is changed for this paper.  Or whether I missed some important settings.

Many thanks for your great work.

**Comment by jbeomlee93** [comment]
Hi @zbf1991 

I think you can check two points now.

1. Your final pseudo label seems weak. Before applying CRFs, I have obtained 69.x mIoU for the pseudo label. I recommend you to check the mIoU of the initial seed (CAM), or IRN hyper-parameters (We slightly changed the original hyper-parameter of the IRN).

2. Please check if you have set up deeplab's initial model correctly. You can download our initial model here: https://drive.google.com/file/d/14soMKDnIZ_crXQTlol9sNHVPozcQQpMn/view

Thanks

**Comment by zbf1991** [comment]
@jbeomlee93 

Thanks for your advice, I re-produced the pseudo label and it reached 69.x%. In order to generate the results reported in your paper, if I need to add CRFs to refine those labels? or I can directly use such labels for the segmentation model? Do you apply CRF to refine them before using them to train deeplab-v2?

**Comment by jbeomlee93** [comment]
Hi @zbf1991 , sorry for the late reply.

I did not find a difference in the segmentation results using the pseudo label with and without applying CRFs, and thus we do not apply CRFs on the pseudo labels.

Thanks.

**Comment by zbf1991** [comment]
Thanks!
